paypay-lab/jp/preauth_capture.json

```diff
@@ -1895,7 +1895,677 @@
                ]
             },
             "model4":{
-               "question":"不正なトークンでアクセスできません",
+               "question":"不正なトークンでAPIにアクセスするときのエラー表示を確認する。",
+               "schema":[
```

Comment:
I think adding new model here may not work
@sureshosur can you review this once you are in?